swagger.yaml

```diff
@@ -116,7 +118,60 @@ paths:
           description: Wrong content type (should be JSON)
         422:
           description: Missing fields
-
+  /projects:
+    get:
+       tags:
+        - projects 
+      summary: Get project list
+      description: Returns a list of project for a specific user 
+      security:
+        - user_id: []
+      responses:
+        200:
+          description: Projects list 
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Projects"
+        404:
+          description: Project not found
+        500:
+          description: Internal server error
+    post:
+      tags:
+        - projects
+      summary: Create a new project
+      security:
+        user_id: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ProjectData"
+      description: Creates a new project
+      responses:
+        201:
+          description: Project created successfully
+        500:
```

Comment:
Missing 415 for invalid mime type, 400 for malformed request and 422 for missing fields, I would also remove the 500. Some earlier endpoint already has the 4XXs added so you can take them from there, they're rather generic.

Reply:
Added